### PR TITLE
Update the information about HEVC support status

### PR DESCRIPTION
**Pull request by StaZhu**
### Description

1. Opera 94+ support HEVC.
2. Add more container that supports HEVC.
3. Browser's HEVC support actually heavily depends on the OS and device,  so update more detail.

### Motivation

Since the support heavily depends on OS and hardware devices due to license reason. We better make developer clear what's the support status of this codec on each platform.

### Additional details

The following is the basis for reference.

Can I Use: [HEVC](https://caniuse.com/hevc)
Chrome Status: [link](https://chromestatus.com/feature/5186511939567616)
enable-chromium-hevc-hardware-decoding: [link](https://github.com/StaZhu/enable-chromium-hevc-hardware-decoding)

### Related issues and pull requests

Relates to #23519

### CODE REVIEW

**Comment by StaZhu** [comment]
Hi reviewer @hamishwillee, please take a look! thanks!

File: files/en-us/web/media/formats/video_codecs/index.md [hunk 1]
```diff
@@ -1066,21 +1066,31 @@ HEVC is a proprietary format and is covered by a number of patents. Licensing is
...
+              <td>94</td>
               <td>11</td>
             </tr>
           </tbody>
         </table>
-        <p>
-          Internet Explorer, Chrome, and Edge only supports HEVC on devices with a
-          hardware codec.
+        <p>Chrome support HEVC for devices with hardware support on Windows 8+, Linux and ChromeOS, for all devices on macOS Big Sur 11+ and Android 5.0+.</p>
+        <p>Edge (Chromium) supports HEVC for devices with hardware support on Windows 10 1709+ when
```
Comment (wbamberg) [inline files/en-us/web/media/formats/video_codecs/index.md:1075]:
```suggestion
        <p>Edge (Chromium) supports HEVC for devices with hardware support on Windows 10 1709+ when
```

File: files/en-us/web/media/formats/video_codecs/index.md [hunk 2]
```diff
@@ -1066,21 +1066,31 @@ HEVC is a proprietary format and is covered by a number of patents. Licensing is
...
               <td>11</td>
             </tr>
           </tbody>
         </table>
-        <p>
-          Internet Explorer, Chrome, and Edge only supports HEVC on devices with a
-          hardware codec.
+        <p>Chrome support HEVC for devices with hardware support on Windows 8+, Linux and ChromeOS, for all devices on macOS Big Sur 11+ and Android 5.0+.</p>
+        <p>Edge (Chromium) supports HEVC for devices with hardware support on Windows 10 1709+ when  
+          <a href="https://apps.microsoft.com/store/detail/hevc-video-extension/9NMZLZ57R3T7">HEVC video extensions from the Microsoft Store</a>
```
Comment (wbamberg) [inline files/en-us/web/media/formats/video_codecs/index.md:1076]:
```suggestion
          <a href="https://apps.microsoft.com/store/detail/hevc-video-extension/9NMZLZ57R3T7">HEVC video extensions from the Microsoft Store</a>
```

**Review by wbamberg — APPROVED** [review]
👍 thank you @StaZhu !

**Pull request merged**